infrastructure-pr-testing.markdown

```diff
@@ -305,24 +358,42 @@ aurora task run -l root <ID> "tail -n1 sw/BUILD/O2-latest/log"
 
 ## Monitoring the checkers
 
+The CI system is monitored using a dedicated
+[Grafana dashboard](https://monit-grafana.cern.ch/d/7_5s-W27k/ci-overview?orgId=65).
+
 Builders are monitored in [Monalisa](http://alimonitor.cern.ch/display?page=github/combined).
 In particular you can use aliendb9 and look at the `github-pr-checker/github.com/github-api-calls`
 metric to know how many API calls are being done by the system.
 
-You can also get a detailed view the activity of the builders in our [Build Infrastructure Cockpit](https://datastudio.google.com/embed/reporting/f41f8c21-c617-4e7e-b14f-0f760c228be4/page/5FCOB). If you are interested in extending the reports, please contact us.
+You can also get a detailed view the activity of the builders in our
+[Build Infrastructure Cockpit](https://datastudio.google.com/embed/reporting/f41f8c21-c617-4e7e-b14f-0f760c228be4/page/5FCOB).
+If you are interested in extending the reports, please contact us.
 
-Some of the CI machines are also included in [IT Grafana dashboard]
-(https://monit-grafana.cern.ch/d/8KnQO4HMk/openstack-vms?orgId=1&var-project_name=ALICE%20Release%20Testing). Notice this includes also machines which do not run CI builds.
+Some of the CI machines are also included in the
+[IT Grafana dashboard](https://monit-grafana.cern.ch/d/8KnQO4HMk/openstack-vms?orgId=1&var-project_name=ALICE%20Release%20Testing).
+Notice this includes also machines which do not run CI builds.
 
 # Troubleshooting
   
 ## Empty logs
 
-Empty logs can happen in the case the build fails in some pre alibuild steps and the harvesting script is not able / is not allowed to fetch logs. In particular, due to the criticality of some operations, logs which might contain sensible information are not retrieved, to avoid exposing them to unprivileged ALICE users. Maintainers of the build infrastructure can follow the instructions in the report to retrieve them.
+Empty logs can happen in the case the build fails in some pre-alibuild steps and
+the harvesting script is not able or not allowed to fetch logs. In particular,
+due to the criticality of some operations, logs which might contain sensitive
+information are not retrieved, to avoid exposing them to unprivileged ALICE
+users. Maintainers of the build infrastructure can follow the instructions in
+the report to retrieve them.
```

Comment:
Please do not split lines. It makes it more difficult to spot differences if a reflow happens.

Reply:
OK, I've undone the splitting.